packages/astro/test/content-collections-render.test.js

```diff
@@ -26,15 +26,15 @@ describe('Content Collections - render()', () => {
 			assert.equal($('ul li').length, 3);
 
 			// Includes styles
-			assert.equal($('link[rel=stylesheet]').length, 2);
+			assert.equal($('link[rel=stylesheet]').length, 1);
```

Comment:
There are all tests that were changed when experimental images were added. This is just changing them back.